models/migrationscripts/updateSchemas20220513.go

```diff
@@ -0,0 +1,72 @@
+package migrationscripts
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/merico-dev/lake/models/migrationscripts/archived"
+	"gorm.io/gorm"
+)
+
+type RefsIssuesDiffs20220513 struct {
+	NewRefId        string `gorm:"primaryKey;type:varchar(255)"`
+	OldRefId        string `gorm:"primaryKey;type:varchar(255)"`
+	NewRefCommitSha string `gorm:"type:varchar(40)"`
+	OldRefCommitSha string `gorm:"type:varchar(40)"`
+	IssueNumber     string `gorm:"type:varchar(255)"`
+	IssueId         string `gorm:"primaryKey;type:varchar(255)"`
+	archived.NoPKModel
+}
+
+func (RefsIssuesDiffs20220513) TableName() string {
+	return "refs_issues_diffs"
+}
+
+type updateSchemas20220513 struct{}
+
+func (*updateSchemas20220513) Up(ctx context.Context, db *gorm.DB) error {
+	cursor, err := db.Model(archived.RefsIssuesDiffs{}).Rows()
+	if err != nil {
+		return err
+	}
+	defer cursor.Close()
+
+	duplicateData := make(map[string]interface{})
+	for cursor.Next() {
+		inputRow := archived.RefsIssuesDiffs{}
+		err := db.ScanRows(cursor, &inputRow)
+		if err != nil {
+			return err
+
+		}
+
+		inputRowkey := fmt.Sprintf("%s%s%s", inputRow.NewRefId, inputRow.OldRefId, inputRow.IssueId)
```

Comment:
`separator` is needed.
consider:
1. `refs/tags/v1.0.0   refs/tags/v2.0.0  12` would become `refs/tags/v1.0.0refs/tags/v2.0.012`
2. `refs/tags/v1.0.0   refs/tags/v2.0.01 2` would become `refs/tags/v1.0.0refs/tags/v2.0.012`